jest.config.js

```diff
@@ -1 +1,12 @@
-module.exports = require('@folio/stripes-acq-components/jest.config');
+const path = require('path');
+const stripesConfig = require('@folio/jest-config-stripes');
+const acqConfig = require('@folio/stripes-acq-components/jest.config');
```

Comment:
Do not make any exceptions to the lint config exported by eslint-config-stripes. Please create a story to remove this and follow the project-wide conventions instead.